### ISSUE TITLE
Cassandra - Invalid list for literal set

### ISSUE THREAD

**Issue by ammills01** [serge-rider/dbeaver#1603]
Using DBeaver to generate an UPDATE script for a Cassandra record puts square brackets on the outside of a set of User Defined Types's (UDT).  When you try to run the update Cassandra throws the error "Invalid list for literal set".  Instead, this should be a curly bracket.  Example:

How it works now:
```
UPDATE Users SET email_addresses = [{username:'user1', domain:'mydomain.com'},{username:
user1', domain:'yourdomain.net'}] WHERE user_id = 1;
```

How it should work:
```
UPDATE Users SET email_addresses = {{username:'user1', domain:'mydomain.com'},{username:
user1', domain:'yourdomain.net'}} WHERE user_id = 1;
```

In the example above the email_addresses column represents a set of EmailAddress (frozen<set<EmailAddress>>), which is a UDT.

**Comment by serge-rider** [comment]
Please post sample DDL.

**Comment by serge-rider** [comment]
Just a reminder

**Comment by ammills01** [comment]
Hope this makes it more clear.  Thanks for looking into it serge.

-- Define the UDT
create type global_transaction_admin.email_address (
    username text,
    domain text
);

-- Create the table with a set of UDT's
create table global_transaction_admin.users (
    user_id int,
    email_addresses set<frozen<email_address>>,
    primary key (user_id)
);

-- Create a single record, did an update here since this is the correct syntax
update global_transaction_admin.users set email_addresses = {{username:'user1', domain:'mydomain.com'},{username:'user1', domain:'yourdomain.net'}} WHERE user_id = 1;

-- Select the record
select * from global_transaction_admin.users;

-- Now, right click the record and go to Generate SQL -> Update.  The code below is produced, which does not work because the [] should be {}.
update global_transaction_admin.users
set email_addresses=[{username:'user1',domain:'mydomain.com'}, {username:'user1',domain:'yourdomain.net'}]
where user_id=1;

**Comment by ammills01** [comment]
Hope this makes it more clear.  Thanks for looking into it serge.

-- Define the UDT
create type email_address (
    username text,
    domain text
);

-- Create the table with a set of UDT's
create table users (
    user_id int,
    email_addresses set<frozen<email_address>>,
    primary key (user_id)
);

-- Create a single record, did an update here since this is the correct syntax
update users set email_addresses = {{username:'user1', domain:'mydomain.com'},{username:'user1', domain:'yourdomain.net'}} WHERE user_id = 1;

-- Select the record
select * from users;

-- Now, right click the record and go to Generate SQL -> Update.  The code below is produced, which does not work because the [] should be {}.
update users
set email_addresses=[{username:'user1',domain:'mydomain.com'}, {username:'user1',domain:'yourdomain.net'}]
where user_id=1;

**Comment by serge-rider** [comment]
Thanks for details. Fix in 4.1.
